tools/topology/topology1/sof-cavs-nocodec.m4

```diff
@@ -47,14 +49,29 @@ define(DMIC16KPROC, `eq-iir-volume')
 define(DMICPROC_FILTER1, `eq_iir_coef_highpass_40hz_20db_48khz.m4')
 define(DMIC16KPROC_FILTER1, `eq_iir_coef_highpass_40hz_20db_16khz.m4')
 
-ifelse(NCORES, `4',
+ifelse(PLATFORM, `tgl',
 `
+define(DMIC_48k_CORE_ID, `0')
+define(DMIC_16k_CORE_ID, `0')
+define(SSP0_CORE_ID, `3')
+define(SSP1_CORE_ID, `2')
+define(SSP2_CORE_ID, `1')
+')
+
```

Comment:
don't we need a block like this here:

ifelse(PLATFORM, `tgl-h',

Reply:
you mean make use of core 0 and core 1 for tgl-h?

Reply:
yes, there's no mapping for tgl-h